src/Symfony/Bundle/FrameworkBundle/Routing/Router.php

```diff
@@ -71,6 +72,14 @@ public function getRouteCollection()
             $this->collection = $this->container->get('routing.loader')->load($this->resource, $this->options['resource_type']);
             $this->resolveParameters($this->collection);
             $this->collection->addResource(new ContainerParametersResource($this->collectedParameters));
+
+            try {
+                $containerFile = ($this->paramFetcher)('kernel.cache_dir').'/'.($this->paramFetcher)('kernel.container_class').'.php';
+                if (file_exists($containerFile)) {
+                    $this->collection->addResource(new FileResource($containerFile));
+                }
```

Comment:
else $this->collection->addResource(new FileExistenceResource($containerFile));

Reply:
Done.